tests/baselines/reference/api/typescript.d.ts

```diff
@@ -4989,7 +4989,7 @@ declare namespace ts {
         readonly modifiers?: NodeArray<Modifier>;
         readonly equalsGreaterThanToken: EqualsGreaterThanToken;
         readonly body: ConciseBody;
-        readonly name: never;
+        readonly name?: never;
```

Comment:
this is basically `name?: undefined` without `exactOptionalPropertyTypes` (and TS codebase doesnt use this flag). Is this property truly optional at runtime anyway? or is it always set to `undefined`?

Reply:
it seems that `createArrowFunction` do not set it at all

Reply:
idk if it is set to undefined somewhere else, I'd say it isn't. Even updateArrowFunction ignores it